client/src/components/MetaTags/MetaTags.js

```diff
@@ -1,5 +1,5 @@
 import Head from 'next/head'
-import Logo from '../../../public/favicon.png'
+import Logo from '../../../public/favicon-32x32.png'
```

Comment:
This is blurry in Chrome, Firefox landing pages on MBP screen and iPhone 12. Is there a way of avoiding that?

Reply:
Changing this to `android-chrome-384x384.png` since it's being used in the opengraph meta-tags and probably displayed at a larger size.